packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6201,6 +6212,17 @@ export default class Meeting extends StatelessWebexPlugin {
           options: {meetingId: this.id},
         });
 
+        // @ts-ignore
+        this.webex.internal.newMetrics.submitClientEvent({
```

Comment:
this is sent basically at the exact same time as "client.share.initiated" from line 6207 above and with the same data, so it feels a bit suspicious if this is really the correct thing to do. Feels like maybe "client.share.initiated" should be sent somewhere earlier, like when the app calls publishTracks() with a screen share track or maybe should be sent from the web app when it actually calls getDisplayMedia()?

Reply:
`client.share.initiated` should probably be sent from somewhere earlier.

Reply:
Worth noting that the 'client.share.initiated' submit was already there before John's changes. @marcin-bazyl do you have a preferred location between your two suggestions?

Reply:
I would say probably better to put it in publishTracks(), because then it will be sent for all other SDK clients too and we won't send it if user clicked "start share" button but then clicked cancel in the browser UI, but ultimately it's the CA team that should tell us when to send it

Reply:
just to add more info for you when you talk to CA team about this, the flow is like this:
1. user clicks "start share"
2. app calls browser API getDisplayMedia()
3. browser displays UI to the user to choose what they want to share - they can also tick a box about audio sharing and they can also just click cancel at this point and the flow ends then
4. app calls SDK's Meeting.publishTracks() with a screen share track
5. SDK gives the track to WCME - WCME will tell Homer that we have a share track for sending
6. SDK asks Locus for the floor via an http request
7. Locus tells us that floor is granted via a Mercury message

If we put 'client.share.initiated' in publishTracks() then that's point 4 in the flow above